webapp/app.py

```diff
@@ -57,6 +58,8 @@ def create_app(testing=False):
     else:
         init_brandstore(app)
 
+    init_db()
```

Comment:
here I think this shouldn't be enabled in the brandstore.
It's a POC but just so we don't forget (you can add a comment or whatever)

Reply:
Moved in the `if`.